### ISSUE TITLE
Lastest release seems to break npx command

### ISSUE THREAD

**Issue by dariye** [intuit/auto#1761]
**Describe the bug**

Running `npx auto shipit` produces an error. My guess is that this release https://github.com/intuit/auto/pull/1752 broke the npx command

**To Reproduce**

1. Run `npx auto shipit` (on CI or local)

**Expected behavior**

- On local, it should create a canary release
- On CI, it should create a regular release

**Screenshots**

<img width="1009" alt="Screenshot 2021-01-28 at 09 47 05" src="https://user-images.githubusercontent.com/3650761/106112809-ff633580-614d-11eb-8932-aa823f2adada.png">

**Environment information:**

```txt
npx: installed 217 in 7.746s
Environment Information:
"auto" version: v10.13.2
"git"  version: v2.25.0
"node" version: v12.16.1
Project Information:
✔ Repository:      pleo-io/deimos (​https://github.com/pleo-io/deimos​)
✔ Author Name:     Andrada-Maria Gae
✔ Author Email:    andrada@pleo.io
✔ Current Version: v17.10.3
✔ Latest Release:  v17.10.3 (​https://github.com/pleo-io/deimos/releases/tag/v17.10.3​)
✖ Labels configured on GitHub project (Try running "auto create-labels")
GitHub Token Information:
✔ Token:            [Token starting with c681]
✔ Repo Permission:  admin
✔ User:             andradaaag
✔ API:              undefined (​undefined​)
✔ Enabled Scopes:   repo, workflow, write:packages
✔ Rate Limit:       4997/5000
```

**Additional context**

**Comment by hipstersmoothie** [comment]
Sorry about that! the next release should fix this

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v10.13.3`](https://github.com/intuit/auto/releases/tag/v10.13.3) :rocket:

**Comment by dariye** [comment]
Thank you @hipstersmoothie
